ci/environment-dev-3.6.yml

```diff
@@ -21,12 +22,12 @@ dependencies:
   - cartopy
   - pip
   - pip:
-    - xskillscore>=0.0.4
+    - xskillscore>=0.0.5
```

Comment:
`xskillscore` is up to 0.0.7 now. Does 0.0.5 cover everything we need here for probabilistic?

Reply:
5 is enough